src/app/withdraw/[txid]/page.tsx

```diff
@@ -0,0 +1,25 @@
+"use server";
+import OldLayout from "../../old-layout";
+import TrackWithdrawalStatus from "./components/withdrawal-status";
+import { getWithdrawalInfo } from "@/actions/get-withdrawal-data";
+
+export default async function WithdrawalStatusPage({
+  params,
+}: {
+  params: Promise<{ txid: string }>;
+}) {
+  const { txid } = await params;
+
+  const result = await getWithdrawalInfo(txid);
```

Comment:
What's the benefit of passing `result` to `TrackWithdrawalStatus`, given we use a `useQuery` there? vs just passing `txid` to `TrackWithdrawalStatus`

Reply:
good question so this is so that the query has initial data indicating whether it should be ran or not basically disabling the query at init and giving the component data on render